packages/wp-source/src/actions.ts

```diff
@@ -20,10 +20,20 @@ const actions: WpSource["actions"]["source"] = {
     // Get route and route params
     const route = normalize(link);
     const routeParams = parse(link);
+    const { query, page } = routeParams;
 
     // Get current data object
     const data = source.data[route];
 
+    // Add the attributes that should be present even if fetch fails or we throw a ServerError below
+    state.source.data[route] = {
+      ...state.source.data[route],
+      link: route,
+      path: routeParams.path,
+      query,
+      page
+    };
```

Comment:
This is the reason that the tests are failing. I'm not 100% sure if we should adjust the test or if this is a problem.

One of the example test cases: 
```
 // Add category to the store
    await store.libraries.source.populate({
      state: store.state,
      response: mockResponse([cat1])
    });
    // Mock Api responses
    api.get = jest.fn().mockResolvedValueOnce(
      mockResponse(cat1PostsPage2, {
        "X-WP-Total": "5",
        "X-WP-TotalPages": "2"
      })
    );
    // Observe changes in isFetching and isReady properties
    const dataState = [];
    observe(() => {
      const { isFetching, isReady } = store.state.source.get(
        "/category/cat-1/page/2/"
      );
      dataState.push({ isFetching, isReady });
    });
    // Fetch entities
    await store.actions.source.fetch("/category/cat-1/page/2/");
    expect(api.get).toHaveBeenCalledTimes(1);
    expect(store.state.source).toMatchSnapshot();
    // Values history of isFetching and isReady
    expect(dataState).toEqual([
      { isFetching: false, isReady: false }, // first values are from a different object
      { isFetching: false, isReady: false }, // initial values from the data object
      { isFetching: true, isReady: false }, // fetch starts
      { isFetching: false, isReady: true } // fetch ends
    ]);
```

Because we don't set the `isFetching` and `isReady` here, the are undefined in the test, after the initial first value are set.

Not sure if we need to set them as well, or if that's fine?